modules/nf-core/comebin/runcomebin/main.nf

```diff
@@ -0,0 +1,71 @@
+process COMEBIN_RUNCOMEBIN {
+    tag "${meta.id}"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/comebin:1.0.4--hdfd78af_0':
+        'biocontainers/comebin:1.0.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(assembly), path(bam, stageAs: "bam/*")
+
+    output:
+    tuple val(meta), path("${prefix}/comebin_res_bins/*.fa.gz"), emit: bins
+    tuple val(meta), path("${prefix}/comebin_res.tsv")         , emit: tsv
+    tuple val(meta), path("${prefix}/comebin.log")             , emit: log
+    tuple val(meta), path("${prefix}/embeddings.tsv")          , emit: embeddings
+    tuple val(meta), path("${prefix}/covembeddings.tsv")       , emit: covembeddings
+    path "versions.yml"                                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args               = task.ext.args ?: ''
+    prefix                 = task.ext.prefix ?: "${meta.id}"
+    def clean_assembly     = assembly.toString() - ~/\.gz$/
+    def decompress_contigs = assembly.toString() == clean_assembly ? "" : "gunzip -q -f $assembly"
+    def cleanup            = decompress_contigs ? "rm ${clean_assembly}" : ""
+    """
+    ${decompress_contigs}
+
+    run_comebin.sh \\
+        -t ${task.cpus} \\
+        -a ${clean_assembly} \\
+        -p bam/ \\
+        -o . \\
```

Comment:
Is there a reason why you don't just pass `${prefix}` to this flag, rather than the `move`?